component-definition.json

```diff
@@ -150,6 +150,37 @@
               }
             }
           }
+        },
+        {
+          "title": "Explore Cards",
+          "id": "explore-cards",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
+                "template": {
+                  "name": "explore-cards",
+                  "model": "explore-cards",
+                  "filter": "explore-cards"
+                }
+              }
+            }
+          }
+        },
+        {
+          "title": "Explore Card",
+          "id": "explore-card",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
```

Comment:
Any reason why this is not a block item but block? Nesting blocks is not recommended!!